[2021-02-28 13:13:34,489] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: bitcoin_price_dag.is_api_available 2021-02-28T12:12:58.544973+00:00 [queued]>
[2021-02-28 13:13:34,493] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: bitcoin_price_dag.is_api_available 2021-02-28T12:12:58.544973+00:00 [queued]>
[2021-02-28 13:13:34,493] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-02-28 13:13:34,493] {taskinstance.py:1043} INFO - Starting attempt 3 of 3
[2021-02-28 13:13:34,493] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-02-28 13:13:34,499] {taskinstance.py:1063} INFO - Executing <Task(HttpSensor): is_api_available> on 2021-02-28T12:12:58.544973+00:00
[2021-02-28 13:13:34,501] {standard_task_runner.py:52} INFO - Started process 40218 to run task
[2021-02-28 13:13:34,508] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'bitcoin_price_dag', 'is_api_available', '2021-02-28T12:12:58.544973+00:00', '--job-id', '17', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/bitcoin_price_dag.py', '--cfg-path', '/var/folders/fh/z0rf64gj7d93005s7pxrkwm40000gn/T/tmpl3q9c8ko', '--error-file', '/var/folders/fh/z0rf64gj7d93005s7pxrkwm40000gn/T/tmp7iu425s9']
[2021-02-28 13:13:34,510] {standard_task_runner.py:77} INFO - Job 17: Subtask is_api_available
[2021-02-28 13:13:34,544] {logging_mixin.py:104} INFO - Running <TaskInstance: bitcoin_price_dag.is_api_available 2021-02-28T12:12:58.544973+00:00 [running]> on host 120.0.168.192.in-addr.arpa
[2021-02-28 13:13:34,568] {taskinstance.py:1255} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=bitcoin_price_dag
AIRFLOW_CTX_TASK_ID=is_api_available
AIRFLOW_CTX_EXECUTION_DATE=2021-02-28T12:12:58.544973+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-02-28T12:12:58.544973+00:00
[2021-02-28 13:13:34,569] {http.py:102} INFO - Poking: ticker
[2021-02-28 13:13:34,572] {base.py:65} INFO - Using connection to: id: bitcoin_api. Host: http://blockchain.info/, Port: None, Schema: , Login: , Password: None, extra: None
[2021-02-28 13:13:34,575] {http.py:140} INFO - Sending 'GET' to url: http://blockchain.info/ticker
[2021-02-28 13:13:34,686] {local_task_job.py:146} INFO - Task exited with return code Negsignal.SIGABRT
